### ISSUE TITLE
AngularJs material-slider does not work within ng-repeat

### ISSUE THREAD

**Issue by sunny9240** [angular/material#391]
material-slider ng-model sets aria-valuenow="NaN"

When slider loads at first time it returns aria-valuenow="NaN" but when i slide values over slider it works properly. can anybody help me out please ?
## material-slider when first time page load 

material-slider class="ng-isolate-scope ng-pristine ng-untouched ng-valid" step="1" ng-model="answer.value" max="70" min="21" flex="" aria-disabled="false" tabindex="0" role="slider" aria-valuenow="NaN" aria-valuemax="70" aria-valuemin="21"

**Comment by Arti3DPlayer** [comment]
Still not works:
```
<md-slider-container flex ng-repeat="n in tpakp track by $index">
                            {{tpakp[$index]}}
                            <md-slider ng-model="tpakp[$index]" ng-change="tpaSliderChange($index)" min="0" max="200" aria-label="{{ 'tpakp'+$index }}" class="md-accent" md-vertical md-range></md-slider>
                        </md-slider-container>
```


![screen shot 2017-05-05 at 13 12 17](https://cloud.githubusercontent.com/assets/3278913/25741676/81e171ae-3194-11e7-8d81-e5c96a76b9bc.png)



```
       "angular": "1.5.9",
        "angular-animate": "1.5.9",
        "angular-aria": "1.5.9",
        "angular-chart.js": "1.1.1",
        "angular-cookies": "1.5.9",
        "angular-material": "1.1.0",
        "angular-messages": "1.5.9",
        "angular-resource": "1.5.9",
        "angular-route": "1.5.9",
```
